tests/demos/test_demos_run.py

```diff
@@ -77,6 +77,7 @@ def py_file(rst_file, tmpdir, monkeypatch):
 
 
 @pytest.mark.skipcomplex  # Will need to add a seperate case for a complex demo.
+@pytest.mark.markif_fixture(pytest.mark.slow, rst_file="linear_wave_equation.py.rst")
```

Comment:
Can you explain this? `rst_file` is not an argument to `test_demo_runs`.

Reply:
This argument is used in the fixture above